apps/web/public/static/locales/de/common.json

```diff
@@ -607,7 +607,7 @@
   "danger_zone": "Achtung",
   "account_deletion_cannot_be_undone": "Vorsicht. Löschen eines Kontos kann nicht rückgängig gemacht werden.",
   "back": "Zurück",
-  "cancel": "Absagen",
+  "cancel": "Abbrechen",
```

Comment:
@mokkin don't both of those words have the same meaning?

Reply:
in that case @PeerRich you might have more context around here

Reply:
here's the problem, we need to introduce two types of "cancel"

because we also use "Cancel" in a different context for "Cancel appointment" which where "Meeting absagen" is correct.

which means we need to introduce a new variable:


"cancel": "Abbrechen",
"cancel_action": "Abbrechen",

and then use the cancel_action in all places where cancel needs to be "Abbrechen"

Reply:
but i think we can merge this, but i will need to watch out where we use "Abbrechen" correclty

Reply:
> here's the problem, we need to introduce two types of "cancel"
> 
> because we also use "Cancel" in a different context for "Cancel appointment" which where "Meeting absagen" is correct.
> 
> which means we need to introduce a new variable:
> 
> "cancel": "Abbrechen", "cancel_action": "Abbrechen",
> 
> and then use the cancel_action in all places where cancel needs to be "Abbrechen"

That's right! The "cancel" I was referring to is the "cancel_action" such as in every dialogue like here in LibreOffice for example at no. 2:
![grafik](https://github.com/calcom/cal.com/assets/2938748/db922f51-68d9-4151-86ae-340ebebbbd6e)